### PR TITLE
Workflow branch

### PR DESCRIPTION
**Pull request by MegumiKim**
Please review my workflow branch to be merged to main. 
 
- All the functions required for unit testing were inaccessible for jest. Issues are opened and communicated to repo's owner. 

- Build and deploy workflow had issues. The page is not built properly. I could not resolve this by myself.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+EMAIL="megumi@noroff.no"
```
Comment (Fermain) [inline .env]: .env should never be tracked and should be in your `.gitignore` file. Instead you commit a `.env.example` file without real credentials.

Comment (Fermain) [reply .env]: I see you do have this in `.gitignore` but you need to delete this file and commit the delete as well.

**Review by Fermain — CHANGES_REQUESTED** [review]
Are you still working on Jest?

The only change I need so far is untracking `.env` and adding `.env.example`.

File: .env (hunk 1 above)
Comment (MegumiKim) [reply .env]: > Should I still write tests, even though I cannot run them??

Comment (MegumiKim) [reply .env]: > Done. Actually, I wanted to use .env to hide credentials in cypress test but I could not import it. Could you tell me how?

Comment (Fermain) [reply .env]: There is an example repo here: https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/server-communication__env-variables

The action happens here: https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/server-communication__env-variables/cypress.config.js

**Pull request closed without merging**